src/main/service/SubscriptionService.ts

```diff
@@ -493,11 +498,7 @@ export class SubscriptionService {
                 listType.jurisdictions.some(jurisdiction => courtJurisdictions.includes(jurisdiction)) &&
                 (listType.restrictedProvenances.length === 0 || listType.restrictedProvenances.includes(userRole))
             ) {
-                if (
-                    selectedListTypes == null ||
-                    selectedListTypes.length == 0 ||
-                    selectedListTypes.includes(listName)
-                ) {
+                if (selectedListTypes != null && selectedListTypes.includes(listName)) {
```

Comment:
Don't need the if/else statements here. The following lines can be simplified to:

listType.checked = selectedListTypes?.length && selectedListTypes.includes(listName);

Reply:
Changed :)